tests/__init__.py

```diff
@@ -327,6 +327,41 @@ def test_addons_env(self):
                 ("test", "-d", "auto/addons/crm/migrations"),
             )
 
+    # HACK https://github.com/itpp-labs/misc-addons/issues/1014
+    # TODO Remove decorator
+    @prerelease_skip
+    def test_addons_env_double(self):
+        """Test double addon reference in addons.yaml"""
+        # Old versions are skiped because they don't support __manifest__.py,
+        # and the test is hacking ODOO_VERSION to pin a commit
+        for sub_env in matrix(odoo_skip={"7.0", "8.0", "9.0"}):
+            sub_env["COMPOSE_FILE"] = "test.yaml"
+            self.compose_test(
+                join(SCAFFOLDINGS_DIR, "addons_env_double"),
+                sub_env,
+                # check module from custom repo pattern
+                ("test", "-d", "custom/src/rma-old"),
+                ("test", "-d", "custom/src/rma-old/rma"),
+                ("test", "!", "-d", "custom/src/rma"),
+                ("test", "!", "-d", "custom/src/rma/rma"),
+                ("test", "-e", "auto/addons/rma"),
+                # Check version is 12.0.1.6.1
+                ("grep", "-Rq", "12.0.1.6.1", "auto/addons/rma/rma"),
+            )
+            sub_env["COMPOSE_FILE"] = "prod.yaml"
+            self.compose_test(
+                join(SCAFFOLDINGS_DIR, "addons_env_double"),
+                sub_env,
+                # check module from custom repo pattern
+                ("test", "-d", "custom/src/rma"),
+                ("test", "-d", "custom/src/rma/rma"),
+                ("test", "!", "-d", "custom/src/rma-old"),
+                ("test", "!", "-d", "custom/src/rma-old/rma"),
+                ("test", "-e", "auto/addons/rma"),
+                # Check version is 12.0.2.0.0
+                ("grep", "-Rq", "12.0.2.0.0", "auto/addons/rma/rma"),
```

Comment:
This test will fail the day 12.0.2.0.0.1 is released. Also on non-v12 builds. Make it more resilient please. Probably, you need to pin the commit for the normal version too in repos.yaml.